### ISSUE TITLE
In Black/White theme high contrast mode, focus is not visible when navigated to ‘Show more’ button in auto suggestions TagPicker.

### ISSUE THREAD

**Issue by shchitg** [microsoft/fluentui#15184]
Browser Details: ​
OS: Windows 10 (Version 1909, Build: 18363.778)
Microsoft Edge Dev Version 84.0.502.0 (Official build) dev (64-bit)

Issue:
In Black/White theme high contrast mode, focus is not visible when navigated to ‘Show more’ button in auto suggestions dropdown.

Please find the below code pen link to repro the issue:

(https://codepen.io/sagsah/pen/RwWxxpw?editors=1010)

Note : this issue is observed in Edge

Actual Steps:
In Black/White theme high contrast mode, focus is not visible when navigated to ‘Show more’ button in auto suggestions dropdown.

Expected Steps:
Focus should be visible when navigated to ‘show more’ button in black/white theme high contrast mode.

Please find the screen shot of the issue 
https://user-images.githubusercontent.com/71761163/94006063-e3750f00-fdbc-11ea-84f2-57a9facf85b3.png

**Comment by paulgildea** [comment]
@shchitg Thanks for the issue!

@xugao just FYI for High Contrast.
